### DISCUSSION TITLE
PlatformIOプロジェクトの削除方法教えて！

### DISCUSSION THREAD

**Discussion by Mihokun** [algyan/XIAOGYAN#66] [General]
【成功 と 今後の心配を...】
* XIAOGYAN、ゲットしたぜ！
* VS Code & PlatformIO 環境構築したぜ！
* HardwareTest のビルドうまく動いたぞ！

けど、なんやかんやいって、いつもこううまくいくとは限らない。

失敗作=いらない プロジェクトの削除の仕方って立派な開発環境になればなるほど、悩む=怖いw

VS Code からプロジェクト選んで削除でいいの？か？
Windowsのエクスプローラから プロジェクト=フォルダ をゴミ箱にポイでいいの？か？

ご存知の方、私にわかりやすいように ^^; 教えてください ^^;

**Comment by matsujirushi** [comment]
> Windowsのエクスプローラから プロジェクト=フォルダ をゴミ箱にポイでいいの？か？

はい、（ほとんどの場合は）それで良いです。
https://community.platformio.org/t/how-to-delete-a-project-without-content/11367/2
